lib/eclint.spec.ts

```diff
@@ -1,6 +1,223 @@
+import gutil = require('gulp-util');
 import common = require('./test-common');
+import eclint = require('./eclint');
+import vfs = require('vinyl-fs');
+import File = require('vinyl');
+import path = require('path');
+
+var expect = common.expect;
 
 // ReSharper disable WrongExpressionStatement
-describe('eclint', () => {
-	// TODO
+describe('eclint gulp plugin', () => {
+	describe('fix file', () => {
+
+		it('fix by default options', (done) => {
+			vfs.src('*.sln*', {
+				stripBOM: false
+			}).pipe(eclint.fix()).on('data', (file: eclint.EditorConfigLintFile) => {
+				expect(file.editorconfig.fixed).to.be.ok;
+				expect(file.editorconfig.errors).to.have.lengthOf(0);
+			}).on('end', () => {
+				done();
```

Comment:
I believe you should only require one `done` in a test. Is `done` ever called twice here? If not, which one is being called? That should be the only `done` in the test. Please do the same throughout the PR.